extensions/amp-analytics/amp-analytics.md

```diff
@@ -425,7 +425,38 @@ This event is also emitted by the document itself and can be configured as:
 "triggers": {
   "renderStart": {
     "on": "render-start",
-    "request": "render-start"
+    "request": "request"
+  }
+}
+```
+
+#### Initial load trigger (`"on": "ini-load"`)
+This event is triggered when initial contents of an element or a document have been loaded.
```

Comment:
All elements support this. Although for some it's somewhat weird. E.g. `amp-pixel`. But for most - it's a good signal and very meaningful.